variant.go

```diff
@@ -3,7 +3,6 @@ package goshopify
 import (
 	"fmt"
 	"time"
-
```

Comment:
This space shouldn't have been removed. Sorry if that was unclear. Please see my other comments. It's a bit pedantic of me to want this space :slightly_smiling_face: